src/movie.c

```diff
@@ -2,31 +2,30 @@
 #include <assert.h>
 #include <stdlib.h>
 
+// Crea una nueva sala de cine.
 movie_t* movie_new(unsigned int cols, unsigned int rows) {
+    // Asigna memoria para los asientos y la estructura de la sala de cine.
     seat_t* seats = (seat_t*)malloc(cols * rows * sizeof(seat_t));
-    if (seats == NULL) {
+    if (seats == NULL)
         return NULL;
```

Comment:
Nunca hagas esto, el hecho que puedas tener bloques if de una sola línea sin corchetes es una aberración, ha habido brechas de seguridad por este tipo de cosas. El cambio está agravado porque en realidad no realizaste ningún cambio real en el código, esto lo único que hace es aumentar las posibilidades que tengas un conflicto con algún compañero.
```suggestion
    if (seats == NULL) {
        return NULL;
    }
```